src/components/MotiDataController/MotiDataController.js

```diff
@@ -83,6 +83,11 @@ var MotiDataController = React.createClass({
     return (
       <div>
         <h3>{this.props.model_id + ' ' + this.props.variable_id + ' ' + this.props.experiment}</h3>
+        <div>
+          <ControlLabel>Download Data &nbsp;</ControlLabel>
```

Comment:
Again, it's preferable to use styling to determine layout. The `&nbsp;` isn't really content, it's a forcing for layout.